views/popups/_all.pug

```diff
@@ -1,5 +1,8 @@
 include load-save
 
+// Must come before the enter-something dialog
```

Comment:
whyso?

Reply:
Awful z-index stuff :) (User is prompted for a name and otherwise that dialog is below the site templates popup)